stdlib/atexit.pyi

```diff
@@ -1,7 +1,10 @@
 from typing import Any, Callable
+from typing_extensions import ParamSpec
+
+_P = ParamSpec("_P")
 
 def _clear() -> None: ...
 def _ncallbacks() -> int: ...
 def _run_exitfuncs() -> None: ...
-def register(func: Callable[..., Any], *args: Any, **kwargs: Any) -> Callable[..., Any]: ...
-def unregister(func: Callable[..., Any]) -> None: ...
+def register(func: Callable[_P, Any], *args: Any, **kwargs: Any) -> Callable[_P, Any]: ...  # type: ignore
+def unregister(func: Callable[_P, Any]) -> None: ...  # type: ignore
```

Comment:
I don't think this makes much sense. `ParamSpec` makes sense in a function definition only when it appears in at least two places, and then it specifies that the two places must match. (This also applies to `TypeVar`.) For `unregister()`, we just want to allow all possible callables, so `Callable[..., Any]` is best.

Reply:
yeah that makes sense, I just thought to use the same type for `register` and then of course for `unregister` 😄